modular_skyrat/master_files/code/modules/client/preferences/middleware/limbs_and_markings.dm

```diff
@@ -176,7 +176,7 @@
 /datum/preference_middleware/limbs_and_markings/proc/set_organ_aug(list/params, mob/user)
 	var/organ_slot = params["organ_slot"]
 	var/augment_name = params["augment_name"]
-	if(augment_name == "Organic")
+	if(augment_name == "Default")
```

Comment:
The `"Default"` should be put in a define, to make it easier for maintainability down the line, and for readability.

Reply:
Also, what kind of impact does renaming this have on existing savefiles that have `"Organic"` rather than `"Default"`?

Reply:
As far as I can tell by reading the code in here on around line 250: `/datum/preference_middleware/limbs_and_markings/get_ui_data(mob/user)`
 It will automatically reset from "Organic" to "Default" simply on the basis that "Organic" is not a valid option anymore.

I was a bit concerned as well, so I've tested extra to verify this and also make sure there are no runtimes.

Reply:
I have quadruple-checked limbs and organs to ensure that they respond OK to my changes, and don't generate any runtimes. I am pretty confident in my test results.

Reply:
Added defines in commit 8593567fe4b61bedf9be1fe5903d22a0288473f8